ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -397,10 +400,11 @@ private void getBlobAndVerify(String blobId, ByteRange range, HttpHeaders expect
     assertEquals("Accept-Ranges not set correctly", "bytes", response.headers().get(RestUtils.Headers.ACCEPT_RANGES));
     byte[] expectedContentArray = expectedContent.array();
     if (range != null) {
-      long blobSize = Long.parseLong(expectedHeaders.get(RestUtils.Headers.BLOB_SIZE));
-      ByteRange resolvedRange = range.toResolvedByteRange(blobSize);
-      assertEquals("Content-Range header not set correctly", RestUtils.buildContentRangeHeader(resolvedRange, blobSize),
+      long blobSize = expectedContent.remaining();
```

Comment:
it seems to me that the previous way of getting the size was the correct one since it refers to the source of truth? this one refers to data that was returned.